README.md

```diff
@@ -22,78 +23,86 @@ Here's a recipe template.
 | Estimated IBU | 30.05 |
 | Estimated Color | ![#000000](https://placehold.it/15/000000/000000?text=+) `#000000` |
 
-### Yeast: Safale S-04
-
-### Fermentables
+### Yeast/Bacteria
+| Type | Name |
+|---|---|
+| Yeast | Safale S-04 |
+| Bacteria | OYL-605 |
 
+### Mash Bill
 | Amount | Fermentable | L° | Bill % |
 |---|---|---|---|
-
-<!-- | 8 lb | Maris Otter | 2.3 | 69.6% |
+<!--
+| 8 lb | Maris Otter | 2.3 | 69.6% |
 | 2 lb | Light Munich | 8 | 17.4% |
 | 12 oz | Caramel / Crystal 80L | 80 | 6.5% |
-| 12 oz | Special Roast | 50 | 6.5% | -->
-
-### Adjuncts
```

Comment:
I think it would be better to have an "Other" category or something. It could include stuff like fining agents, things added during fermentation like fruit or sugar, and water.